src/appengine/handlers/upload_testcase.py

```diff
@@ -392,7 +392,7 @@ def do_post(self):
         'trustedAgreement') == TRUSTED_AGREEMENT_TEXT.strip()
 
     if (not trusted_agreement_signed and
```

Comment:
I'm not sure I like this.
1. Won't it be an issue if we have linux jobs that are trusted?
2. I think is_remotely_executing_utasks is returning the wrong result and should be fixed instead.

Reply:
1. We don't have linux jobs that are trusted at the moment, right? It would need updating later on for sure - esp once we get hardware
2. hmm do you think linux platform check should live within is_remotely_executing_utasks?

Reply:
chatted offline - to make this clearly temporary and submit for now until we have more robust configs set up